sources/packages/backend/apps/api/src/route-controllers/supporting-user/supporting-user.aest.controller.ts

```diff
@@ -28,12 +28,12 @@ export class SupportingUserAESTController {
    * an application id for AEST user.
    * @param applicationId application id.
    * @return list of supporting users of an
-   * application, i.e ApplicationSupportingUsersApiOutDTO
+   * application, i.e ApplicationSupportingUsersAPIOutDTO
```

Comment:
We can remove the DTO names `ApplicationSupportingUsersAPIOutDTO` in the comment.